### PR TITLE
fixed ck-editor internal link visualization

### PR DESCRIPTION
**Pull request by marcelmoosbrugger**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | none
| License | MIT
| Documentation PR | none

#### What's in this PR?

In the ck-editor internal links to pages which are in draft mode but already have a published version got marked as unpublished. This PR fixes that issue. Moreover it's fixed that the CK-Editor removes whitepsaces before the link.

### CODE REVIEW

**Comment by danrot** [comment]
Changelog missing :)

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/ckeditor-internal-link/main.js [hunk 1]
```diff
@@ -212,7 +212,7 @@ define(['underscore', 'text!./form.html'], function(_, formTemplate) {
 
                                 this.href = id;
                                 this.hrefTitle = title;
-                                this.hrefPublished = item.publishedState;
+                                this.hrefPublished = !!item.published;
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/ckeditor-internal-link/main.js]: This seems to work initially, but when reloading the page, the same error still persists.

**Comment by danrot** [comment]
Changelog missing

**Comment by marcelmoosbrugger** [comment]
@danrot The issues you mentioned in your comments should be fixed now.

**Comment by danrot** [comment]
Looks good, waiting for tests.

**Pull request merged**